server.js

```diff
@@ -1,5 +1,5 @@
 var http = require('http'),
-    configuration = require('./configuration.js'),
```

Comment:
would love if you also include parsing this file in Configurator somewhere. It will be useful to be: DEFAULT < FILE < ENV < CL arguments